### PR TITLE
Digitalocean sponsorship

### PR DESCRIPTION
**Pull request by robacarp**
While lurking around during the setup of the freenode irc channel, for which I am very grateful, I was surprised to find out that Amber is not currently sponsored in hosting. Specifically some server space was needed at the time to get the gitter-irc bot hosting in place, but that may be covered now.

With some light discussion I took some initiative to ask some friends for tips on getting DigitalOcean sponsorship and yesterday I got a positive reply back. We can discuss the details of the sponsorship in chat, but I wanted to submit this pull request as one suggestion for how their logo could look on the amberframework.org site. I'm open to suggestions on placement, [color](https://www.digitalocean.com/company/logos-and-badges/), syntax, etc. I'm a little rusty with my front end experience, and I'm still getting used to slang.

Here is a rendered screenshot: 

image

### CODE REVIEW

**Comment by eliasjpr** [comment]
Lets chat about the location of the logo. See this as an example http://thetheme.io/thesaas/block-partner.html

**Comment by eliasjpr** [comment]
I would like (If it is possible) for us to add a post on DO docs on how to setup amber on Digital Ocean. We already support deploy to DO out of the box.

**Comment by robacarp** [comment]
@eliasjpr I tinkered with the partner blocks provided by theSaaS and came up with this. Adding an additional section below the `Community Driven` section makes the White area long, but changing the background to grey made the section blend into the footer, which looked worse. So I left it white. 

Open for suggestions, let me know what you think.

image

**Comment by elorest** [comment]
It would be awesome to be sponsored by DO or even Vultr whom I recently started using as well. Seems that they're trying to be DO in a way to might be willing to compete for sponsorship.

**Comment by robacarp** [comment]
@amberframework/core Digitalocean has granted generous sponsorship. What's needed to get this out so Amber is holding up its end of the bargain?

**Comment by robacarp** [comment]
Lets try that again: @amberframework/core-team

File: src/views/layouts/_footer.slang [hunk 1]
```diff
@@ -13,7 +13,11 @@ footer.site-footer.py-30.bg-gray
             | Amber Framework
           | . All rights reserved.
         br
-        a.github-button href="https://github.com/amberframework/amber" data-icon="octicon-star" data-size="large" data-show-count="true" aria-label="Star amberframework/amber on GitHub" Star
+
+        .row
+          .col-6.col-md-6.col-lg-4
+            a.github-button href="https://github.com/amberframework/amber" data-icon="octicon-star" data-size="large" data-show-count="true" aria-label="Star amberframework/amber on GitHub" Star
```
Comment (marksiemers) [inline src/views/layouts/_footer.slang]: The latest release of slang should allow this syntax:
```slim
.row: .col-6.col-md-6.col-lg-4
  a.github-button href="https://github.com/amberframework/amber" data-icon="octicon-star" data-size="large" data-show-count="true" aria-label="Star amberframework/amber on GitHub" Star
```

@amberframework/core-team - WDYT?

**Pull request merged**